chapters/research-design.tex

```diff
@@ -1,288 +1,586 @@
-Depending on the distribution of results,
-estimates may become completely uninformative
-with relatively little loss to follow-up.
-
-The diff-in-diff model is a four-way comparison.\sidenote{\url{https://dimewiki.worldbank.org/wiki/ieddtab}}
-The experimental design intends to compare treatment to control,
-after taking out the pre-levels for both.\sidenote{\url{https://www.princeton.edu/~otorres/DID101.pdf}}
-Therefore the model includes a time period indicator,
-a treatment group indicator (the pre-treatment control is the base level),
-and it is the \textit{interaction} of treatment and time indicators
-that we interpret as the differential effect of the treatment assignment.
-
-\subsection{Regression discontinuity}
-
-\textbf{Regression discontinuity (RD)} designs differ from other RCTs
-\index{regression discontinuity}
-in that the treatment group is not directly randomly assigned,
-even though it is often applied in the context of a specific experiment.\sidenote{
-\url{https://dimewiki.worldbank.org/wiki/Regression_Discontinuity}}
-(In practice, many RDs are quasi-experimental, but this section
-will treat them as though they are designed by the researcher.)
-In an RD design, there is a \textbf{running variable}
-which gives eligible people access to some program,
-and a strict cutoff determines who is included.\cite{lee2010regression}
-This is usually justified by budget limitations.
-The running variable should not be the outcome of interest,
-and while it can be time, that may require additional modeling assumptions.
-Those who qualify are given the intervention and those who don't are not;
-this process substitutes for explicit randomization.\sidenote{\url{http://blogs.worldbank.org/impactevaluations/regression-discontinuity-porn}}
-
-For example, imagine that there is a strict income cutoff created
-for a program that subsidizes some educational resources.
-Here, income is the running variable.
-The intuition is that the people who are ``barely eligible''
-should not in reality be very different from those who are ``barely ineligible'',
-and that resulting differences between them at measurement
-are therefore due to the intervention or program.\cite{imbens2008regression}
-For the modeling component, the \textbf{bandwidth},
-or the size of the window around the cutoff to use,
-has to be decided and tested against various options for robustness.
-The rest of the model depends largely on the design and execution of the experiment.
-
-%------------------------------------------------
-
-\section{Quasi-experimental designs}
+The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Randomized_Control_Trials}}
+  \index{randomized control trials}
+In randomized control trials, the control group is randomized --
```

Comment:
This is perhaps just a matter of perspective, but i think this is more intuitive from the opposite angle, i.e. to say: the 'treatment' being studied is randomly allocated-- that is, from an eligible population, a random subset of units are given access to the treatment. The non-selected units serve as a counterfactual.

Reply:
I think you are right that it is more intuitive the other way, but I described it this way because I think it is important to take a moment to understand that this is _why_. I want people to stop and think while reading this section -- thinking of it as selecting the counterfactual at random from the eligible is a nice brain stretch IMO